### ISSUE TITLE
Incorrect fsf address in header files

### ISSUE THREAD

**Issue by sanjayankur31** [ibancg/lingot#56]
Hi there,

Fedora maintainer for lingot here. I was just updating to the latest release 1.1.1, and rpmlint pointed out that the FSF address in the headers is wrong:

```
lingot-devel.x86_64: E: incorrect-fsf-address /usr/include/lingot/lingot-audio-alsa.h
lingot-devel.x86_64: E: incorrect-fsf-address /usr/include/lingot/lingot-audio-jack.h
lingot-devel.x86_64: E: incorrect-fsf-address /usr/include/lingot/lingot-audio-oss.h
lingot-devel.x86_64: E: incorrect-fsf-address /usr/include/lingot/lingot-audio-pulseaudio.h
lingot-devel.x86_64: E: incorrect-fsf-address /usr/include/lingot/lingot-audio.h
lingot-devel.x86_64: E: incorrect-fsf-address /usr/include/lingot/lingot-config-scale.h
lingot-devel.x86_64: E: incorrect-fsf-address /usr/include/lingot/lingot-config.h
lingot-devel.x86_64: E: incorrect-fsf-address /usr/include/lingot/lingot-core.h
lingot-devel.x86_64: E: incorrect-fsf-address /usr/include/lingot/lingot-defs.h
lingot-devel.x86_64: E: incorrect-fsf-address /usr/include/lingot/lingot-filter.h
lingot-devel.x86_64: E: incorrect-fsf-address /usr/include/lingot/lingot-msg.h

```

Could you please take a look when you have a minute and update it?

**Comment by ibancg** [comment]
Hi Ankur, thanks for pointing it out.

It should be fixed after #57, already merged into `master`. I hope the wording is correct, as I don't have those lint checks to test it myself.

I think I will roll it out as release `1.1.2`.

**Comment by sanjayankur31** [comment]
Thanks for the quick reply @ibancg . Looks good, but I'll try to find the time to make a temporary package from the current snapshot and run the linter on it this week.
